event_sale_reservation/__manifest__.py

```diff
@@ -4,19 +4,17 @@
 {
     "name": "Sell event reservations",
     "summary": "Allow selling event registrations before the event exists",
-    "version": "15.0.1.0.0",
+    "version": "17.0.1.0.0",
     "development_status": "Beta",
     "category": "Marketing",
     "website": "https://github.com/OCA/event",
     "author": "Tecnativa, Odoo Community Association (OCA)",
-    "maintainers": ["Yajo"],
+    "maintainers": ["pilarvargas-tecnativa"],
     "license": "AGPL-3",
     "application": False,
     "installable": True,
     "depends": [
         "event_sale",
-        "web_ir_actions_act_multi",
```

Comment:
ir.actions.act_multi was being used to chain several actions that odoo didn't have, in this module close one wizard and open another. This is no longer necessary in this module as a wizard closes automatically when doing any interaction. That way when you open the next wizard you don't have to tell it to close it because odoo already closes it. In the same way, the method that returned a reload action has been removed because when the last wizard is closed, the order view is reloaded.

How do you think it should be reflected in OU?